Sources/SwiftDocC/Infrastructure/DocumentationConverter.swift

```diff
@@ -250,13 +257,22 @@ public struct DocumentationConverter: DocumentationConverterProtocol {
         // Copy images, sample files, and other static assets.
         try outputConsumer.consume(assetsInBundle: bundle)
         
+        let symbolIdentifiersMeetingRequirementsForExpandedDocumentation: [String]? = symbolIdentifiersWithExpandedDocumentation?.compactMap { (identifier, accessControlLevelRequirement) -> String? in
```

Comment:
I believe we need to consider whether the symbol is underscored and if it's SPI here as well.

We could leave it a TODO to pass in those requirements later-on and hard-code for them now but it might be nice to just pass them in.

Eventually I'll imagine us needing to support the recently added `@_documentation(visibility: ...)` modifier as well.